Lib/doctest.py

```diff
@@ -1627,6 +1637,15 @@ def merge(self, other):
             d[name] = (failures, tries, skips)
 
 
+def _n_items(items: list) -> str:
+    """
+    Helper to pluralise the number of items in a list.
+    """
+    n = len(items)
+    s = "" if n == 1 else "s"
+    return f"{n} item{s}"
```

Comment:
```suggestion
    return f"{n} item{pl(n)}"
```
Reject if seems ugly to you.